bin/rialto/runtime/src/millau.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
+// This file is part of Parity Bridges Common.
+
+// Parity Bridges Common is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity Bridges Common is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges Common.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Configuration parameters for the Millau Substrate chain.
+
+use bp_rialto::Header;
+use pallet_substrate_bridge::AuthoritySet;
+use sp_core::crypto::Public;
+use sp_finality_grandpa::AuthorityId;
+use sp_std::vec;
+
+/// The first header known to the pallet.
+///
+/// Note that this does not need to be the genesis header of the Millau
+/// chain since the pallet may start at any arbitrary header.
+pub fn initial_header() -> Header {
+	Header {
+		parent_hash: Default::default(),
+		number: Default::default(),
+		state_root: Default::default(),
+		extrinsics_root: Default::default(),
```

Comment:
```suggestion
		state_root: hex!("756ba147f373121b1c26281c51ec914d3f2790a064ad7728c2fa2935d0afe893").into(),
		extrinsics_root: hex!("03170a2e7597b7b7e3d84c05391d139a62b157e78786d8c082f29dcf4c111314").into(),
```

Reply:
(please check this, though - maybe I have some other changes that affect this state in #398)

Reply:
I got the same `extrinsics_root` but a different `state_root`. What I'm going to do is leave my values for this PR and you can change them to ones that work in #398